cmd/podman/images/pull.go

```diff
@@ -154,15 +154,20 @@ func imagePull(cmd *cobra.Command, args []string) error {
 	}
 	// Let's do all the remaining Yoga in the API to prevent us from
 	// scattering logic across (too) many parts of the code.
-	pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), args[0], pullOptions.ImagePullOptions)
-	if err != nil {
-		return err
+	pulledImages := make([]string, len(args))
+	for i, arg := range args {
+		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
+		if err != nil {
+			return err
+		}
+		pulledImages[i] = pullReport.Images[0]
```

Comment:
`pullReport.Images` can have more than one image (see `podman pull -a`) so we should do as follows:
```go
var pulledImages []string
for i, arg := .... {
   ....
   pulledImages = append(pulledImages, pullReport.Images...)
}
```
This will append all pulled images.

```